tests/components/tibber/test_services.py

```diff
@@ -246,6 +249,90 @@ async def test_get_prices_start_tomorrow(
     }
 
 
+async def test_get_prices_with_timezone(
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test __get_prices with start date tomorrow."""
+    freezer.move_to(STARTTIME)
+    call = ServiceCall(
+        DOMAIN, PRICE_SERVICE_NAME, {"start": dt_util.start_of_local_day().isoformat()}
+    )
+
+    result = await __get_prices(call, hass=create_mock_hass())
+
+    assert result == {
+        "prices": {
+            "first_home": [
+                {
+                    "start_time": STARTTIME,
+                    "price": 0.46914,
+                    "level": "VERY_EXPENSIVE",
+                },
+                {
+                    "start_time": STARTTIME + dt.timedelta(hours=1),
+                    "price": 0.46914,
+                    "level": "VERY_EXPENSIVE",
+                },
+            ],
+            "second_home": [
+                {
+                    "start_time": STARTTIME,
+                    "price": 0.46914,
+                    "level": "VERY_EXPENSIVE",
+                },
+                {
+                    "start_time": STARTTIME + dt.timedelta(hours=1),
+                    "price": 0.46914,
+                    "level": "VERY_EXPENSIVE",
+                },
+            ],
+        }
+    }
+
+
+async def test_get_prices_without_timezone(
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test __get_prices with start date tomorrow."""
+    freezer.move_to(STARTTIME)
+    call = ServiceCall(
+        DOMAIN,
+        PRICE_SERVICE_NAME,
+        {"start": dt_util.start_of_local_day().replace(tzinfo=None).isoformat()},
+    )
+
+    result = await __get_prices(call, hass=create_mock_hass())
+
+    assert result == {
+        "prices": {
+            "first_home": [
+                {
+                    "start_time": STARTTIME,
+                    "price": 0.46914,
+                    "level": "VERY_EXPENSIVE",
+                },
+                {
+                    "start_time": STARTTIME + dt.timedelta(hours=1),
+                    "price": 0.46914,
+                    "level": "VERY_EXPENSIVE",
+                },
+            ],
+            "second_home": [
+                {
+                    "start_time": STARTTIME,
+                    "price": 0.46914,
+                    "level": "VERY_EXPENSIVE",
+                },
+                {
+                    "start_time": STARTTIME + dt.timedelta(hours=1),
+                    "price": 0.46914,
+                    "level": "VERY_EXPENSIVE",
+                },
+            ],
+        }
+    }
```

Comment:
Create a parameterized test out of it as only the `start` data param of the service call is changing